services/QuillLMS/app/workers/populate_evidence_activity_healths_worker.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class PopulateEvidenceActivityHealthsWorker
```

Comment:
IMO having two files with only one letter - `s` - difference may be confusing. What about naming this something like `PopulateEvidenceActivityHealthAggregateWorker`?